src/components/app/Home.css

```diff
@@ -88,23 +88,29 @@
   transition: background-color 300ms;
 }
 
+/* Overcome the selector specificity of the default focus styling by including the tag. */
+a.homeSectionButton:focus,
+button.homeSectionButton:focus {
+  border-radius: 0;
+  box-shadow: var(--photon-focus-shadow);
+  outline: 0;
+}
+
 .homeSectionLoadProfile {
   position: relative;
 }
 
 .homeSectionActionButtons {
-  display: flex;
+  display: grid;
```

Comment:
I made this have a gap between:

<img width="515" alt="image" src="https://user-images.githubusercontent.com/1588648/98381853-12171400-2010-11eb-92b8-84decb3cd12d.png">

<img width="548" alt="image" src="https://user-images.githubusercontent.com/1588648/98381891-193e2200-2010-11eb-9b37-39a8f949f429.png">

Reply:
yeah this is better!

Reply:
You could have kept this a flex, and use `gap` instead of `grid-gap`. Actually `grid-gap` is the old name, it's probably better to use `gap` also for grid. (see https://developer.mozilla.org/en-US/docs/Web/CSS/gap)

So my suggestion would be to do this: keep this flex but with `gap: 10px`, for simplicity.